src/main/webapp/jsp/instructorFeedbackAjaxRemindParticularStudentsModal.jsp

```diff
@@ -2,14 +2,50 @@
 <%@ page import="teammates.common.util.Const"%>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 
-<c:forEach items="${data.responseStatus.noResponse}" var="userToRemindEmail">
-    <div class="checkbox">
-        <label>
-            <input type="checkbox" name="<%= Const.ParamsNames.SUBMISSION_REMIND_USERLIST %>" value="${userToRemindEmail}">
-            ${data.responseStatus.emailNameTable[userToRemindEmail]}
-        </label>
-    </div>
-</c:forEach>
+<table class="table-responsive table table-striped table-bordered">
+    <tr>
+        <th class="student_remind_tick"></th>
```

Comment:
Should CSS for this purpose be applied on this header cell instead of directly on the checkboxes?